src/test/java/com/microsoft/sqlserver/jdbc/AlwaysEncrypted/JDBCEncryptionDecryptionTest.java

```diff
@@ -91,18 +92,15 @@ public void testJksName(String serverName, String url, String protocol) throws E
      */
     @ParameterizedTest
     @MethodSource("enclaveParams")
+    @Tag(Constants.reqExternalSetup)
     public void testAkvName(String serverName, String url, String protocol) throws Exception {
         setAEConnectionString(serverName, url, protocol);
 
-        try {
```

Comment:
Please do not remove this test, it is still valid.

Reply:
this is not removed, it's been renamed to testAkvNameWithAuthCallback